packages/svelte/package.json

```diff
@@ -56,6 +56,9 @@
       "types": "./types/index.d.ts",
       "import": "./src/runtime/store/index.js"
     },
+    "./tag-version": {
```

Comment:
Should we name this `/internal/tag-version` instead to better signal that you shouldn't use this?